contracts/contracts/sbtc-registry.clar

```diff
@@ -56,6 +56,11 @@
 ;; stored to avoid replay
 (define-map multi-sig-address principal bool)
 
+;; Data structure to store the active protocol contracts
+(define-map protocol-contracts principal bool)
+(map-set protocol-contracts .sbtc-bootstrap-signers true)
+(map-set protocol-contracts .sbtc-deposit true)
```

Comment:
What do you think about adding something like:

```clarity
(if (not is-in-mainnet) (map-set protocol-contracts tx-sender true) true)
```

That way, we can call protocol functions directly in unit tests.

Reply:
Yeap, makes sense. Addressed.